core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1375,6 +1375,54 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(500, consumer0.committed(tp2).offset)
   }
 
+  @Test
+  def testPerPartitionLeadMetricsCleanUpWithSubscribe() {
+    val numMessages = 1000
+    val topic2 = "topic2"
+    createTopic(topic2, 2, serverCount)
```

Comment:
It seems that this is already done in setup()?

Reply:
@junrao `setup` in BaseConsumerTest only creates one topic named "topic". This case tests how lead metrics will be calculated when consumer subscribed two topics, so a second topic was created in the test case itself, like what all other cases did.